src/Api/Controllers/CollectionsController.cs

```diff
@@ -108,21 +113,28 @@ public async Task<IEnumerable<SelectionReadOnlyResponseModel>> GetUsers(string o
         public async Task<CollectionResponseModel> Post(string orgId, [FromBody]CollectionRequestModel model)
         {
             var orgIdGuid = new Guid(orgId);
-            if (!await ManageAnyCollections(orgIdGuid))
+            var collection = model.ToCollection(orgIdGuid);
+
+            if (!await CanCreateCollection(orgIdGuid, collection.Id) &&
+                !await CanEditCollectionAsync(orgIdGuid, collection.Id))
```

Comment:
Hmm, do we have scenarios where we can create collections with only edit permission?

Reply:
No, This is actually checking if it's a create operation and you have permission or if it's an edit operation and you have permission.

The not logic makes it a little more confusing.